docs/api/saved-objects/bulk_get.asciidoc

```diff
@@ -11,36 +11,44 @@ experimental[] Retrieves multiple {kib} saved objects by ID.
 
 `POST /api/saved_objects/_bulk_get`
 
+`POST /s/<space_id>/api/saved_objects/_bulk_get`
+
+[[saved-objects-api-bulk-get-path-params]]
+==== Path parameters
+
+`space_id`::
+  (Optional, string) An identifier for the space. If `space_id` is not provided in the URL the default space is used.
+
 [[saved-objects-api-bulk-get-request-body]]
 ==== Request Body
 
 `type`::
   (Required, string) Valid options include `visualization`, `dashboard`, `search`, `index-pattern`, `config`, and `timelion-sheet`.
 
 `id`::
-  (Required, string) ID of the retrieved object.
+  (Required, string) ID of the retrieved object. This can be a Kibana's unique identifier or a custom one.
```

Comment:
I would change the second sentence to:

`The ID includes the {kib} unique identifier or a custom identifier.`